src/server/api/campaign-contact.js

```diff
@@ -93,31 +109,20 @@ export const resolvers = {
             formatted[responseId]['interaction_step_id'] = interactionStepId
           }
         } else {
-          const answerOption = ""
-          const answerOptions = []
-          answerOptions.push({
-            "value": answerValue, 
-            "interaction_step_id": interactionStepId
-          })
-          const campaignId = results[i].left.campaign_id
-          const createdAt = results[i].left.created_at
-          const questionStepId = results[i].left.interaction_step_id
-          const questionStepParentInteractionId = results[i].right.parent_interaction_id
-          const question = results[i].left.question
-          const script = results[i].left.script
-
           formatted[responseId] = {
             "contact_response_value": responseValue,
             "interaction_step_id": interactionStepId,
```

Comment:
The id of the interaction step with the answer_option that has the same value as the question_response.